src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -413,6 +413,10 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Condition="'$(ThreadPoolMaxThreads)' != ''"
                                     Value="$(ThreadPoolMaxThreads)" />
 
+    <RuntimeHostConfigurationOption Include="System.StartupHookProvider.IsSupported"
```

Comment:
(nit) the rest of the `RuntimeHostConfigurationOption` items are in alphabetical order on the `Include`. This makes it easier to find the switch you are looking for. Can you move this new item up between the `System.Runtime` and `System.Text` entries?